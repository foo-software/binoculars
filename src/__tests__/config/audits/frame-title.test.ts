import auditFrameTitle from '../../../config/audits/frame-title';

describe('frame-title audit', () => {
  it('should match snapshot', () => {
    expect(auditFrameTitle('en')).toMatchSnapshot();
  });
});
