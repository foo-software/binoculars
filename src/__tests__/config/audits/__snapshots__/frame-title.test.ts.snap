// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`frame-title audit should match snapshot 1`] = `
Object {
  "description": "Screen reader users rely on frame titles to describe the contents of frames. [Learn more](https://web.dev/frame-title/).",
  "title": "\`<frame>\` or \`<iframe>\` elements have a title",
}
`;
